integrations/gcp/main.py

```diff
@@ -130,7 +131,17 @@ async def feed_events_callback(request: Request) -> Response:
             f"Couldn't find project ancestor to asset {asset_name}. Other types of ancestors and not supported yet."
         )
     except ResourceNotFoundError:
-        logger.exception(f"Didn't find any {asset_type} resource named: {asset_name}")
+        logger.warning(
+            f"Didn't find any {asset_type} resource named: {asset_name}. Deleting ocean entity."
+        )
+        await ocean.unregister(
+            [
+                Entity(
+                    blueprint=asset_type,
+                    identifier=asset_name,
+                )
+            ]
+        )
```

Comment:
I think you don't need to do this with @omby8888 feature of delete on event, please verify